apps/server/src/modules/board/service/board-do-authorizable.service.ts

```diff
@@ -28,16 +29,16 @@ export class BoardDoAuthorizableService implements AuthorizationLoaderService {
 	}
 
 	async getBoardAuthorizable(boardDo: AnyBoardDo): Promise<BoardDoAuthorizable> {
-		const rootBoardDo = await this.getRootBoardDo(boardDo);
+		const rootDo = await this.getRootBoardDo(boardDo);
 		const parentDo = await this.getParentDo(boardDo);
```

Comment:
// TODO parentDo maybe is not needed on each boardDo - currently only for SubmissionItems, so, we could spare an extra query for all other cases